src/cookbooks/bigquery/access.md

```diff
@@ -1,82 +1,79 @@
 
-You can locate a list of supported BigQuery client libraries [here](https://cloud.google.com/bigquery/docs/reference/libraries).
-
-Detailed REST reference can be found [here](https://cloud.google.com/bigquery/docs/reference/rest/).
+> This method requires [BigQuery Access](#BigQuery-Access-Request) to be provisioned.
 
-## From `bq` Command-line Tool
+Steps to use:
 
 - Install the [GCP SDK](https://cloud.google.com/sdk/)
 - Authorize `gcloud` with either your user account or provisioned service account. See documentation [here](https://cloud.google.com/sdk/docs/authorizing).
-    - `gcloud auth login`
+  - `gcloud auth login`
 - Set your google project to your team project
-    - `gcloud config set project moz-fx-data-bq-<team-name>`
-    - project name will be provided for you when your account is provisioned.
+  - `gcloud config set project moz-fx-data-bq-<team-name>`
+  - project name will be provided for you when your account is provisioned.
+
+Once configured, you can now simply use the `bq` command-line client. Here's a quick one
+which lists the tables and views in a BigQuery dataset:
 
-### `bq` Examples
-List tables and views in a BigQuery dataset
-``` bash
+```bash
 bq ls moz-fx-data-derived-datasets:telemetry
 ```
-Query a table or view
- ``` bash
- bq query --nouse_legacy_sql 'select count(*) from `moz-fx-data-derived-datasets.telemetry.main` where submission_date = "2019-08-22" LIMIT 10'
- ```
 
-Additional examples and documentation can be found [here](https://cloud.google.com/bigquery/docs/bq-command-line-tool).
+And here's another which gets the count of entries in `telemetry.main` on `2019-08-22` in the nightly channel:
+
+```bash
+bq query --nouse_legacy_sql 'select count(*) from `moz-fx-data-derived-datasets.telemetry.main` where submission_date = "2019-08-22" and normalized_channel="nightly"'
+```
+
+Additional examples and documentation can be found [in the BigQuery command-line reference](https://cloud.google.com/bigquery/docs/bq-command-line-tool).
+
+### API Access
 
-### From client SDKs
+> This method requires [BigQuery Access](#BigQuery-Access-Request) to be provisioned.
 
-Client SDKs for various programming languages do not access credentials the
-same way as the `gcloud` and `bq` command-line tools. The client SDKs
-generally assume that the machine is configured with a service account and
-will look for JSON-based credentials in several well-known locations rather
-than looking for user credentials.
+For advanced use cases involving programmatic access -- including automated workloads, ETL, [BigQuery Storage API](https://cloud.google.com/bigquery/docs/reference/storage/).
+
+You can locate a list of supported BigQuery client libraries [here](https://cloud.google.com/bigquery/docs/reference/libraries).
+
+Although usually you will want to use a client library, Google also provides a [detailed reference of their underlying REST API](https://cloud.google.com/bigquery/docs/reference/rest/).
+
+#### Service Accounts
```

Comment:
This should really be a seperate cookbook, see https://bugzilla.mozilla.org/show_bug.cgi?id=1631909